.gitattributes

```diff
@@ -23,3 +23,4 @@
 *.png git-encoding=BINARY working-tree-encoding=BINARY
 *.gif git-encoding=BINARY working-tree-encoding=BINARY
 *.zip git-encoding=BINARY working-tree-encoding=BINARY
+*.blob git-encoding=BINARY working-tree-encoding=BINARY
```

Comment:
This should not specify encodings for binary files, and the list should be sorted:
```
*.blob binary
*.gif binary
*.jpg binary
*.png binary
*.zip binary
```
See https://git-scm.com/book/en/v2/Customizing-Git-Git-Attributes.